### PR TITLE
PLT-1259 Converted TeamMembersModal to React-Bootstrap

### PR DESCRIPTION
**Pull request by hmhealey**
Needs to be rebased after #1485 is in. To see the changes for this PR, look at d0cace9

### CODE REVIEW

**Comment by coreyhulen** [comment]
@hmhealey needs rebase

**Comment by coreyhulen** [comment]
+1

File: web/react/components/member_list_team.jsx [hunk 1]
```diff
@@ -2,17 +2,45 @@
...
+    componentDidMount() {
+        UserStore.addChangeListener(this.onChange);
+    }
+
+    componentWillUnmount() {
+        UserStore.removeChangeListener(this.onChange);
+    }
+
+    onChange() {
+        this.forceUpdate();
```
Comment (crspeller) [inline web/react/components/member_list_team.jsx]: This is not really how this should be handled. forceUpdate should be avoided.

**Comment by hmhealey** [comment]
@crspeller I've moved the users list into the component's state instead of just constructing it in render. Reacy for review again

**Pull request merged**